docs/troubleshooting.md

```diff
@@ -5,20 +5,19 @@ This page contains a few commonly occurring questions.
 ## Container keeps pending/ terminating
 
 1. Check the output of `kubectl get nodes`
-2. Check the output of `kubectl describe pod <your-pod>`
```

Comment:
`kubectl describe pod <your-pod>` can help check whether the pod is scheduled successfully.